url/resources/a-element-origin.js

```diff
@@ -5,7 +5,6 @@ function setBase(base) {
 }
 
 function bURL(url, base) {
-  base = base || "about:blank"
```

Comment:
I think removing this line broke a bunch of tests? I haven't dug into the new failures but e.g. https://github.com/web-platform-tests/wpt/pull/29579/checks?check_run_id=2991897363 seems bad.

Reply:
Thanks for the follow-up. I've pushed a further fix, let me know if that seems better here.